app/support/products/scheduling_support.rb

```diff
@@ -155,14 +155,19 @@ class ReservationFinder
     attr_accessor :time, :rule, :day_start, :day_end, :options
 
     def initialize(time, rule, options = {})
-      self.time = time
+      self.time = rule.instrument.cutoff_hours > 0 ? time_with_cutoff_hours(time, rule.instrument) : time
```

Comment:
Do we actually need this `> 0` check? If we used `to_i`, then `nil` and zero would covert to `0`.